src/bin/travis/resources/centaur.wdl

```diff
@@ -22,7 +22,7 @@ task centaur {
 
     runtime {
         docker: "geoffjentry/centaur-cromwell:latest" 
```

Comment:
No. When we first did it I asked if people cared and they said no.

What's way worse than this is that the encrypted bundle for travis, the original files live & knowledge of how to build it are only on my laptop